admin-dev/themes/new-theme/scss/pages/_orders.scss

```diff
@@ -40,6 +40,14 @@
   }
 }
 
+#addProductTableRow,
+.editProductRow {
+  // div selector is important here, it is used for empty labels regular labels must not be hidden
+  div.form-control-label {
```

Comment:
Ping @NeOMakinG

Reply:
Yes, thats because in the twig template, empty labels are replaced with an empty div with this class I think the purpose is to keep the element aligned in a regular form, even when some of the form inputs have no labels

Reply:
Weird to add an empty div :(

Reply:
Could you wrap these inside stylelint comments like this ?

```
/* stylelint-disable */
  div.form-control-label {
   ...
  }
  /* stylelint-enable */
```